scripts/dotnet-cli-build/SharedFrameworkTargets.cs

```diff
@@ -33,7 +39,22 @@ public static BuildTargetResult PublishSharedFramework(BuildTargetContext c)
             // We publish to a sub folder of the PublishRoot so tools like heat and zip can generate folder structures easier.
             string SharedFrameworkNameAndVersionRoot = Path.Combine(SharedFrameworkPublishRoot, "shared", SharedFrameworkName, SharedFrameworkNugetVersion);
 
-            DotNetCli.Stage0.Publish("--output", SharedFrameworkNameAndVersionRoot, SharedFrameworkSourceRoot).Execute().EnsureSuccessful();
+            string publishFramework = "dnxcore50"; // Temporary, use "netcoreapp" when we update nuget.
+            string publishRuntime;
+            if (PlatformServices.Default.Runtime.OperatingSystemPlatform == Platform.Windows)
+            {
+                publishRuntime = $"win7-{PlatformServices.Default.Runtime.RuntimeArchitecture}";
+            }
+            else
+            {
+                publishRuntime = PlatformServices.Default.Runtime.GetRuntimeIdentifier();
+            }
+
+            DotNetCli.Stage0.Publish(
```

Comment:
I think the original idea was to stay independent of the rest of the build (so that you could just run the package targets), but since we are pulling corehost from the build as well that seems unimportant now. I switched it.